webpack/components/TabbedView/TabbedView.js

```diff
@@ -4,22 +4,30 @@ import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
 import './TabbedView.scss';
 
 const TabbedView = ({ tabs }) => {
-  const [activeTabKey, setActiveTabKey] = useState(0);
-  const handleTabClick = (_event, tabIndex) => setActiveTabKey(tabIndex);
+  const [activeTabKey, setActiveTabKey] = useState('details');
+  const [openedTabs, setOpenedTabs] = useState([]);
+  const handleTabClick = (_event, tabIndex) => {
+    setOpenedTabs(origTabs => [...origTabs, tabIndex]);
+    setActiveTabKey(tabIndex);
+  };
 
   return (
     <Tabs activeKey={activeTabKey} onSelect={handleTabClick}>
-      {tabs.map((tab, i) => {
+      {tabs.map((tab) => {
         const { title, content } = tab;
+        const lowerCaseTitle = title.toLowerCase();
+        // Load if tab is clicked on and keep loaded to prevent re-loading tab content.
```

Comment:
Ya I wasn't sure exactly how we should handle this, I do see on our existing pages we load the content when the tab is clicked, so loading everything at once will be a change, and I'm not sure if there are performance implications with users who have a lot of data. On the other hand it does pre-emptively load the tabs so it could be an improvement in the UX.

Lets discuss with UX!